### ISSUE TITLE
Reset to default button

### ISSUE THREAD

**Issue by adiraoco** [siteorigin/settings#17]
I'm not sure if this is possible, but a reset to default button would be nice. :)

**Comment by Misplon** [comment]
We've wanted this in the past. Ages ago @gregpriday hacked a solution for the Customizer. One of the WP updates broke it so we left it alone. @gregpriday want to have another look? I can log.

**Comment by gregpriday** [comment]
Yeah, I'll take another look. Will try do it in a less hacky way. @Misplon - it's already logged ;)
